docs/UserGuide.md

```diff
@@ -314,6 +314,21 @@ The following formats are valid for entering day values:
 | Saturday    | `Saturday`, `Sat`                  |
 | Sunday      | `Sunday`, `Sun`                    |
 
+The formats are case-insensitive, i.e. both `Monday` and `mon` are valid.
+
+The date and time formats accepted by ModBook use the symbols in the table below:
+
+| Symbol | Meaning                                 |
+| ------ | --------------------------------------- |
+| H      | Hour of 24-hour time. Goes from 0 to 23 |
+| h      | Hour of 12-hour time. Goes from 1 to 12 |
+| m      | Minute                                  |
+| a      | AM/PM indicator                         |
+| L      | Month as text                           |
+| M      | Month as number                         |
+| y      | Year                                    |
+| d      | day of month                            |
+
```

Comment:
u moved this table to before the date table cause it reads better this way i suppose?

Reply:
ya